tests/test_pdf.py

```diff
@@ -72,7 +72,7 @@ def test_pdf_basicapi_tests(backend):
     pars = pdf.config.suggested_init()
     data = source['bindata']['data'] + pdf.config.auxdata
 
-    tensorlib, _ = backend
+    tensorlib, _ = pyhf.get_backend()
```

Comment:
this should not have been changed. We have to inject the backend this way for testing purposes (if the tests are in weird orders, the `pyhf.get_backend()` isn't properly reset for tests).